ui/src/components/Patients.tsx

```diff
@@ -113,10 +120,30 @@ const Patient: React.FC = () => {
         </Route>
         <Route exact path={match.path}>
           <div>
-            <button className="flex justify-center items-center space-x-2 px-4 py-2 rounded-lg border-black border-2 bg-blue text-white">
-              <Share />
-              <div> Refer Patient </div>
-            </button>
+            <ChoiceModal className="flex flex-col"
+                         choice={Main.Provider.Provider.CreateReferral}
+                         contract={pcpContract?.contractId}
+                         submitTitle="Create Referral"
+                         buttonTitle="Refer Patient"
+                         icon={<Share />}
+                         initialValues={ { policy: Nothing,
+                                           receiver: Nothing,
+                                           encounterId: Nothing,
+                                           procedureCode: Nothing,
+                                           diagnosisCode: Nothing,
+                                           siteServiceCode: Nothing,
+                                           appointmentPriority: Nothing } } >
```

Comment:
```suggestion
                         initialValues={ {
                           policy: Nothing,
                           receiver: Nothing,
                           encounterId: Nothing,
                           procedureCode: Nothing,
                           diagnosisCode: Nothing,
                           siteServiceCode: Nothing,
                           appointmentPriority: Nothing,
                         } } >
```
I read that trailing commas are now a thing, so maybe this is more ideomatic?